.sass-lint.yml

```diff
@@ -93,11 +74,11 @@ rules:
   empty-args: 0
   hex-length: 2
   hex-notation: 2
-  # indentation: 2
+  indentation: 0
   leading-zero: 2
   nesting-depth:
     - 2
-    - max-depth: 4
+    - max-depth: 5
```

Comment:
why?

Reply:
it's my personal experience. I increase it sometimes when need to theme multiple levels menu

Reply:
i would better change it to 3 levels actually. less chaos